articles/systems/01-camera-system.md

```diff
@@ -1,23 +1,44 @@
-# Headline
+# The Camera System
 
-A brief overview of the tutorial.
+The camera system is the entry point for any platform and sets up the working environment required for that platform.
 
-## What you'll need
+This is broken up in to THREE distinct components:
 
-List previous steps if needed and a short list of required materials or skills.
+## Mixed Reality Root configuration system
 
-## Getting Started
+In the Root configuration, the specific implementation for the camera system is selected, which defaults to the XRTK implementation for the camera system.  In most cases this does not need to be touched, however, advanced users can replace it with their own system if they wish.
 
-1. Step 1
-2. Step 2
+![](../../images/Configuration/CameraSystem/CameraSystemProfile.png)
+
+## Camera System platform configuration
+
+The Camera system for the XRTK allows for different implementations to be utilized for different runtime or build platforms. Each platform provided by the XRTK includes a default camera system configuration for that platform which can be customized if you wish:
+
+![](../../images/Configuration/CameraSystem/CameraSystemSettings.png)
+
+The configuration holds:
+
+* The default configuration - a Camera profile to be used where none is specified, a fallback default
+* Additional Camera Profile variants, which specifies the Name, Type, Camera Profile and platforms this variant is meant to run on.
+
+> For more detail on the [Platforms](08-platform-framework.md) that are available to configure, see the [XRTK Platform Framework](08-platform-framework.md) documentation.
```

Comment:
Why is `08-platform-framework.md` linked twice?

Reply:
More the merrier 😉